src/applications/vaos/components/NewTabAnchor.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function NewTabAnchor({ href, 'aria-describedby': describedBy, ...props }) {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      aria-describedby={
+        describedBy ? `${describedBy} ${'new-tab-msg-1'}` : 'new-tab-msg-1'
+      }
+      {...props}
+    >
+      {props.children}
+      <img
```

Comment:
Perhaps you could use font awesome vs an image here for the same look. I think its lighter weight too `<i class="fa fa-link"></I>`